src/components/side-panel/source-data-catalog.js

```diff
@@ -55,15 +55,23 @@ const DatasetTitle = styled.div`
     }
   }
 `;
+
 const DatasetTagWrapper = styled.div`
   display: flex;
   color: ${props => props.theme.textColor};
   font-size: 11px;
   letter-spacing: 0.2px;
-  
+  overflow: auto;
+
   .dataset-color {
     margin-top: 5px;
```

Comment:
can you add `flex-shrink: 0;` so the square doesn't get squeezed?

Reply:
Certainly. I'll add that property once I'm home from work. Thanks for the response @heshan0131.

Reply:
Hey @heshan0131 where exactly do you want me to place this? The comment is on lien 65, but I didn't add any of that code.

Reply:
To ‘.dataset-color’

Reply:
Gotcha, thanks. Just updated that & pushed.